<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialBox</class>
 <widget class="QDialog" name="SerialBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>526</y>
     <width>276</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serial_status">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>DISCONNECTED</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="port_name_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>96</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Port Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="baud_rate_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>96</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Baud Rate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serial_combo_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtTop</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>476</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="button_layout">
    <item>
     <widget class="QPushButton" name="disconnect_button">
      <property name="text">
       <string>Disconnect</string>
      </property>
      <property name="autoDefault">
        <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connect_button">
      <property name="text">
       <string>Connect</string>
      </property>
      <property name="autoDefault">
        <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="baud_combo_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtTop</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="serial_display">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>351</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="left_slider">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>278</y>
     <width>128</width>
     <height>30</height>
    </rect>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="right_slider">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>278</y>
     <width>128</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="up_slider">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>145</y>
     <width>30</width>
     <height>128</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="down_slider">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>313</y>
     <width>30</width>
     <height>128</height>
    </rect>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
